Specs/Number/Korean/NumberModel.json

```diff
@@ -2595,11 +2582,9 @@
     ]
   },
   {
-    "Input": "양말 한 다스",
+    "Input": "계란 한 다스만 사주시겠어요?",
```

Comment:
Could you add some comments in the PR discussion on such changes?
The whitespace/comma is easy to grasp, but these sentence structure changes would benefit from details on what was incorrect in the original text.

Reply:
These cases are part of 22 new distinct cases (identified and shared with you earlier) which required restructure with language expert.
Updated PR description accordingly.

Reply:
I understand that overall reason. But it would be nice to have some of the linguistic details documented in the PR discussion for future reference. 
Documents/reports don't go along with the repo code, so it would be harder for others to track offline.
It doesn't need to be very detailed, but comments on the specifics of the sentence structure would be helpful (in the PR, not in the code/specs for these cases).